src/protojure/internal/grpc/client/providers/http2/jetty.clj

```diff
@@ -142,26 +148,38 @@
    (transmit-data-frame stream data false 0))
   ([^Stream stream data last? padding]
    (stream-log :trace stream "Sending DATA-FRAME with " (count data) " bytes, ENDFRAME=" last?)
-   @(jetty-callback-promise
-     (fn [cb]
-       (let [frame (DataFrame. (.getId stream) (ByteBuffer/wrap data) last? padding)]
-         (.data stream frame cb))))))
+   (-> @(jetty-callback-promise
+         (fn [cb]
+           (let [frame (DataFrame. (.getId stream) (ByteBuffer/wrap data) last? padding)]
+             (.data stream frame cb))))
+       (p/catch
```

Comment:
I think this can go-away once the deref is in the proper place.

Reply:
I think it was unrelated to the deref -- cruft from the earlier cancel promise iteration